docs/mllib-optimization.md

```diff
@@ -203,6 +203,9 @@ regularization, as well as L2 regularizer.
 recommended.
 * `maxNumIterations` is the maximal number of iterations that L-BFGS can be run.
 * `regParam` is the regularization parameter when using regularization.
+* `convergenceTol` designates the end of iterations based on relative comparison
```

Comment:
If linking to Breeze, would be better to link straight to its LBFGS. However I don't know if we want to promise that implementation detail. 

Should we also be updating the javadoc for `LBFGS.scala` and other similar places where this param is documented?

I'm not sure this text helps someone understand what the value is or how to set it -- what are adjusted values? You're right, it's fairly opaque beyond being something relative (rather than something that might need to change with the input). The thing to know is that lower values mean more iterations and the value must be nonnegative. So how about something more like:

Controls how much relative change is still allowed when L-BFGS is considered to converge. Must be nonnegative. Lower values are less tolerant and therefore generally cause more iterations to be run.